magma/magma
# fix(nms): If an organization is deleted also delete its users

## Summary

Also the delete the users belonging to the organization when the organization is deleted.

- fixes #12720

## Test Plan

* Create an organization 
* Adds some users to the organization
* Delete the organization
* Log in to the NMS database with `docker-compose exec postgres psql -U nms`
* Check that the users have been removed with `select * from "Users";`
